v2/devdoc/logging.md

```diff
@@ -238,6 +238,12 @@ void a(LOG_CONTEXT_HANDLE log_context, const char* correlation_id)
 
 Note: When logging with local contexts, the context passed to `LOGGER_LOG` has to be passed using the address of the context (&).
 
+If too many properties are passed when defining local contexts, the error shall be indicated by calling a log internal error report function.
+
+### LOG_ABORT_ON_ERROR
+
+The internal error report function shall abort the execution if `LOG_ABORT_ON_ERROR` is defined so that such error cases can easily be noticed during development.
```

Comment:
> during development [](http://example.com/codeflow?start=142&length=19)

Assuming that this would also happen in production which would need to be dealt with.

Reply:
The problem is the following: Let's imagine using some fixed size arrays to store data because we're building some information on the stack (context or even doing an snprintf as we do today).
There is always the possibility that the information does not fit.
For a string the answer is simple: let's truncate it. Even then there is some loss of information that can be overseen easily.
Same applies for these contexts. If too many properties are passed in then we'd have to ditch information.

Then my thinking is: great let's know about it in our development so we can address it (either increase the logging array sizes or anything else). 

In production logging will still be best effort -  if too many properties are passed, they'll be dropped.